dags/defaults.py

```diff
@@ -77,41 +77,50 @@
 stages:
   canary:
   - selectors:
-    - assignment: unassigned
-      owner: DFINITY
-      status: Healthy
-      nodes_per_group: 1
+      join:
+      - assignment: unassigned
+        owner: DFINITY
+        status: Healthy
+        nodes_per_group: 1
+      - assignment: API boundary
+        owner: DFINITY
+        status: Healthy
+        nodes_per_group: 1
```

Comment:
Let's put this with the last "assigned" batch in the canary stage (lines 106 etc.).